core/domain/suggestion_registry.py

```diff
@@ -511,6 +560,27 @@ def __init__( # pylint: disable=super-init-not-called
         self.score_category = score_category
         self.last_updated = last_updated
 
+    @classmethod
+    def from_dict(cls, suggestion_dict):
+        """Return a Domain object for a suggestion of type
+        SUGGESTION_TYPE_ADD_QUESTION from a dict.
+
+        Args:
+            suggestion_dict: dict. The dict representation of Suggestion object.
+
+        Returns:
+            SuggestionAddQuestion. The corresponding suggestion domain
+            object.
+        """
+        return cls(
+            suggestion_dict['suggestion_id'], suggestion_dict['target_id'],
+            suggestion_dict['target_version_at_submission'],
+            suggestion_dict['status'], user_services.get_user_id_from_username(
+                suggestion_dict['author_name']),
```

Comment:
Why is this method nedded (I don't think its a good plan to use service here as well as return user_id in the dict.)

Reply:
removed